kotest-extensions/kotest-extensions-htmlreporter/build.gradle.kts

```diff
@@ -0,0 +1,71 @@
+plugins {
+   id("java")
+   kotlin("multiplatform")
+   id("java-library")
+   id("org.jetbrains.kotlin.plugin.spring") version Libs.kotlinVersion
```

Comment:
Is spring needed ?

Reply:
I don't believe so, this must be a typo. I'll check it and remove it.